packages/table/src/common/_loading.scss

```diff
@@ -0,0 +1,42 @@
+// Copyright 2017 Palantir Technologies, Inc. All rights reserved.
```

Comment:
@themadcreator what do you think of this file living in `common/` dir? 
it's not a true common file because it defines CSS rules (not just sass stuff). but it affects both cells and headers.